CITATION.cff

```diff
@@ -0,0 +1,35 @@
+cff-version: 1.2.0
+message: If you use this software, please cite it as below.
+type: software
+title: Snakemake
+version: 8.29.3
+date-released: '2025-03-09'
+# doi: 10.5066/??
+abstract: The Snakemake workflow management system is a tool to create
+  reproducible and scalable data analyses.
+repository-artifact: https://pypi.org/project/snakemake/
+repository-code: https://github.com/snakemake/snakemake
+license: MIT
+authors:
+- family-names: Köster
+  given-names: Johannes
+  orcid: https://orcid.org/0000-0001-9818-9320
+  alias: johanneskoester
```

Comment:
It does not feel right to have me as only author here. Can we scrape this info from git/github and update the citation.cff automatically like every week or so in a CI job? For simplicity, as this is yaml, the ci job you use https://yte-template-engine.github.io/ for templating, basically get contributors with pygithub and insert them here sorted by number of commits or so.
Also the release and the date could be inserted automatically like that.

Reply:
Thought further about this. I think the update should be done directly in .github/workflows/release-please.yml.
Whenever a new release is created.

Reply:
I.e. no weekly job needed, just upon every release. If you want, this could even be a generic new github action, so that it can be applied in all other snakemake related repos and by any project outside. Suprising that there is no such action yet, seems like a common problem to me. But there are seemingly only citation.cff validators so far.